ui/components/YamlView.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+import styled from "styled-components";
+
+type Props = {
+  className?: string;
+  yaml: string;
+}
+
+function YamlView({ yaml, className }: Props) {
+  return (
+    <pre className={className}>
+      <code>
+        {yaml}
+      </code>
+    </pre>
+  );
+}
+
+export default styled(YamlView).attrs({
+  className: YamlView.name,
+})`
+  font-size: medium;
```

Comment:
It shrinks the text from 14px to 12px - possibly we just haven't reset `<pre>` tags somewhere?

Before I added it, it looked like:
![Screenshot from 2022-06-06 15-20-36](https://user-images.githubusercontent.com/1743/172210827-4db2feb0-5278-4a56-b919-ca9420f24b8f.png)
The screenshot doesn't quite do justice to the experience of seeing the content being bigger than the headers.

Reply:
I didn't realize that `medium` was actually a real option hahaha - it might make sense to change it to `${props => props.theme.fontSizes.small}` but it's also fine as is

Reply:
That's better :+1: